keps/sig-security/3203-auto-refreshing-official-cve-feed/README.md

```diff
@@ -0,0 +1,690 @@
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Story 1
+
+As a K8s end user, I want a list of CVEs with relevant information that I can fetch programmatically, so I can understand when new CVEs are announced
+
+#### Story 2
+
+As a K8s maintainer, I want to create a process that auto-updates CVE feed, when SRC announces new CVEs such that I do not have to do extra work to maintain this feed manually
+
+## Proposal {#proposal}
+
+### Pre-requisites
+- [x] https://github.com/kubernetes/test-infra/pull/23428
+- [x] Search and Identify closed issues that have a CVE ID e.g. CVE-1001-12345 in the issue description or summary (This search [filter](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+in%3Abody+%22CVSS%3A3.%22+label%3Acommittee%2Fsecurity-response+is%3Aclosed+) is giving the most accurate data so far)
+- [x] Label those issues with `official-cve-feed` using https://docs.github.com/en/rest/reference/issues REST API
+- [x] https://github.com/kubernetes/committee-security-response/pull/133
+
+
+### Overview
+
+- Generate a JSON document using the results from the filtered label on `k/k` repo.
+- Create a Prow job to periodically generate this JSON document.
+- Update the JSON doc when needed (e.g. when a new CVE is announced) in `k/website`
+- Using Hugo, publish the list from this JSON document on official k8s website
+
+### Risks and Mitigations
+
+#### JSON blob construction will fail
+
+If this happens, we expect the job too fail. If blob construction fails, the failure will alert the owners of this feature and we will take action as needed. If the failure can not be fixed in a reasonable amount of time, the CVE feed will be stale until it is fixed. In case of urgency from community to get the refreshed feed, JSON blob will be manually updated via usual PR review and approval process.
+
+#### Urgent CVE feed refresh
+
+In some extenuating circumstances, we may need to update the CVE feed within minutes of the official CVE announcement, instead of waiting for the merge based or periodical website rebuild. In those situations, manual updates to JSON blob using usual PR reviews and approval process can be implemented.
+
+#### Misuse of Auto-Refresh feature
+
+Without proper filtering and control over who can label GitHub issues, the list of CVEs can become a list with poor signal to noise ratio making the list unusable. 
+
+For this purpose, the filtering is applied such that only issues that are marked as `closed`
+will be part of the list. Also, additionally, the `official-cve-feed` label is a 
+[restricted](https://github.com/kubernetes/test-infra/blob/master/config/prow/plugins.yaml#L140-L150)
+label that can only be applied by SRC and SIG Security Tooling Leads.
```

Comment:
Maybe add some text about why closed issues are the right ones to list.
(I'd expect to mention all labelled issues, closed or not, provided that they are publicly visible).

Reply:
My assumption is that open SRC created GitHub issues with a CVE ID that are not under embargo, are unlikely. Maybe worth adding that as an assumption as a clarification?